cmd/build-packages.go

```diff
@@ -75,5 +76,12 @@ func init() {
 		"Build without docker; this may adversely affect your system.  Only supported on Linux, and requires CAP_SYS_ADMIN.",
 	)
 
+	buildPackagesCmd.PersistentFlags().StringP(
```

Comment:
We need to throw an error early if this is not supplied; otherwise the issue is hard to track down.  (It shows up as an invalid docker container name because it starts with `-`)  Bonus points for actually checking that the image exists, I guess?